Simplified/NYPLSettingsAccountDetailViewController.m

```diff
@@ -402,6 +402,12 @@ - (void)scanLibraryCard
   }];
 }
 
+#pragma mark - NYPLUserAccountProvider Protocol
+
+- (BOOL)isSignedIn {
```

Comment:
this and the NYPLSignedInStateProvider compliance at the top can also be removed I think